app/managers.py

```diff
@@ -0,0 +1,48 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(
+            self,
+            first_name: str,
+            last_name: str
+    ) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name}"
+            " (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> None:
```

Comment:
this function return list with data not None, it must be `def all(self) -> list[Actor]:`